.github/common_environment.yml

```diff
@@ -5,7 +5,7 @@
   SLACK_FOOTER:          School Experience
   DOCKER_REPOSITORY:     ghcr.io/dfe-digital/schools-experience
   DOMAIN:                london.cloudapps.digital
```

Comment:
I believe this variable is not is use anymore

Reply:
will be removed in another PR